### DISCUSSION TITLE
option to use configuration properties while jans installation

### DISCUSSION THREAD

**Discussion by manojs1978** [JanssenProject/jans#1592] [Q&A] [answered]
when we are installing jans server we need to enter some information manually at command prompt.
can we have option to use property file with install.py or with rpm to install jans in silent mode?

ex: pytho3 install.py -f propertyfile
jans-1.0.0-el8.x86_64.rpm -f propertyfile

**Comment by ossdhaval** [comment]
I am not sure if the feature already exists, but it'll be valuable.
cc: @devrimyatar

**Comment by manojs1978** [comment, marked as answer]
yes we can use property file as below.
sudo python3 install.py --args="-f setup.properties -c -n
